kernel/Makefile

```diff
@@ -214,12 +214,22 @@ endef
 # Build Targets
 # Debug targets only for latest stable and LTS stable
 #
+ifeq ($(ARCH), x86_64)
```

Comment:
Just to double check (because I trip over this sometimes), is the whitespace here significant, i.e. the comparison is with `" x86_64"` including the space and unless `$(ARCH)` has the same space it wont match? I may be mis-remembering this though. I guess you will find out when building...

Also, would it be worth keeping the common (i.e. default) kernel(s) common and only splitting out some of the optional extras? Might be clearer?

Reply:
I copy and pasted the conditional from further up in the Makefile. I've now tidied it up (it did work before and now as well).

I've also followed your suggested in keeping the common kernels common. I suspect once s390x support lands more of the kernels will be moved to arch specific clauses, but that's fine